react/features/base/responsive-ui/actions.js

```diff
@@ -53,6 +70,7 @@ export function clientResized(clientWidth: number, clientHeight: number) {
                 clientWidth: availableWidth
             });
             dispatch(setAspectRatio(clientWidth, clientHeight));
+            width && dispatch(setFilmstripWidth(width));
```

Comment:
I'm wondering  if we can move the whole filmstrip logic from here to a middleware in features/filmstrip? WDYT? Maybe it will fit better there?